### PR TITLE
#6 add more validations for the config file

### PR DESCRIPTION
**Pull request by gluckzhang**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -1,56 +1,67 @@
...
+    return config
+
 
-    # config validation
+def validate_configs(config):
     if config["group_by.enabled"]:
         if len(config["group_by.groups"]) < 1 or len(config["group_by.groups"]) > 2:
-            logging.error(
-                "If group_by is enabled, there should be at leaest one group, and at most two groups!")
+            logging.error("If group_by is enabled, there should be at leaest one group, and at most two groups!")
```
Comment (Ebaneck) [inline main.py:37]:
```suggestion
            logging.error("If group_by is enabled, there should be at least one group, and at most two groups!")
```

File: main.py [hunk 2]
```diff
@@ -1,56 +1,67 @@
...
 
     if len(config["target_aws_accounts"]) == 0:
-        logging.error(
-            "There should be at leaest one target AWS accounts defined in the config!")
+        logging.error("There should be at leaest one target AWS accounts defined in the config!")
         sys.exit(1)
 
     labels = config["target_aws_accounts"][0].keys()
+    if group_label_names and (group_label_names & set(labels)):
+        logging.error("Some of label names in group_by are the same as AWS account labels!")
```
Comment (Ebaneck) [inline main.py:53]:
```suggestion
        logging.error("Some label names in group_by are the same as AWS account labels!")
```

**Review by Ebaneck — COMMENTED** [review]
LGTM, just a few typos here and there!

**Comment by gluckzhang** [comment]
> LGTM, just a few typos here and there!

Perfect. Thank you @Ebaneck :)

**Pull request merged**